wordpress-mobile/WordPress-Android
# Issue/4438 pn read signal clear all

Fixes #4438

To test:
This needs the backend counterpart to test, but basically:

1) log in with your account on an iOS and an Android device, as well as the web client on a browser.
2) have some other account like/comment/etc one of your posts
3) you'd receive push notifications about this action on both your iOS and Android devices - don't open them or do anything there yet, just observe this.
4) open the Notifications TAB on the web client
5) the notifications on both iOS and Android devices should be cleared.


Needs review: @kwonye
